packages/pharos/src/components/coach-mark/pharos-coach-mark.scss

```diff
@@ -0,0 +1,183 @@
+  --coach-mark-text-size: var(--pharos-font-size-small);
+  --coach-mark-heading-color: var(--pharos-color-marble-gray-20);
+  --coach-mark-text-color: var(--pharos-color-marble-gray-20);
+  --coach-mark-text-spacing: var(--pharos-spacing-1-x);
+  --coach-mark-pip-size: 10px;
+
+  display: block;
+  position: absolute;
+}
+
+:host([variant='dark']) {
+  --coach-mark-background-color: var(--pharos-button-color-base-overlay-background-base);
+  --coach-mark-border-color: var(--coach-mark-background-color);
+  --coach-mark-heading-color: var(--pharos-color-white);
+  --coach-mark-text-color: var(--pharos-color-marble-gray-80);
+}
+
+.coach-mark {
+  animation-duration: 1000ms; /* Delay duration of 0ms, fade duration of --pharos-transition-duration-default */
+  animation-name: fadein;
+  animation-timing-function: ease-in-out;
+
+  &[aria-hidden='true'] {
+    display: none;
+    opacity: 0;
+  }
+
+  &.delay-short {
+    animation-duration: 1600ms; /* Delay duration of 600ms, fade duration of --pharos-transition-duration-default */
+    animation-name: fadein-short;
+  }
+
+  &.delay-long {
+    animation-duration: 3000ms; /* Delay duration of 2000ms, fade duration of --pharos-transition-duration-default */
+    animation-name: fadein-long;
+  }
+}
+
+.coach-mark__content {
+  position: relative;
+  background-color: var(--coach-mark-background-color);
+  border: var(--coach-mark-border-width) solid var(--coach-mark-border-color);
+  padding: var(--pharos-spacing-one-half-x) var(--pharos-spacing-three-quarters-x);
+  min-width: 100px;
+  width: fit-content;
+  font-size: var(--coach-mark-text-size);
+  color: var(--coach-mark-text-color);
+  line-height: var(--coach-mark-text-spacing);
+
+  &::before,
+  &::after {
+    content: '';
+    background-color: var(--coach-mark-background-color);
+    position: absolute;
+    top: var(--pip-top);
+    left: var(--pip-left);
+    height: var(--coach-mark-pip-size);
+    width: var(--coach-mark-pip-size);
+    transform: translate(-50%, -50%) rotate(45deg);
+    z-index: 1;
+  }
+
+  &::after {
+    --coach-mark-border-width-x2: calc(var(--coach-mark-border-width) * 2);
+
+    background-color: var(--coach-mark-border-color);
+    height: calc(var(--coach-mark-pip-size) + var(--coach-mark-border-width-x2));
+    width: calc(var(--coach-mark-pip-size) + var(--coach-mark-border-width-x2));
+    z-index: -1;
+  }
+}
+
+.coach-mark__heading {
+  color: var(--coach-mark-heading-color);
+  padding-right: var(--pharos-spacing-one-and-a-half-x);
+}
+
+.coach-mark__close {
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 1;
+}
+
+.coach-mark__wrapper {
+  height: fit-content;
+  width: fit-content;
+}
+
+.coach-mark-side__right .coach-mark__content {
+  --pip-left: 0;
+}
+
+.coach-mark-side__left .coach-mark__content {
+  --pip-left: 100%;
+}
+
+.coach-mark-side__top .coach-mark__content {
+  --pip-top: 100%;
+}
+
+.coach-mark-side__bottom .coach-mark__content {
+  --pip-top: 0;
+}
```

Comment:
With BEM we might also consider something like `.coach-mark__side--right` since the "side" feels like the element and _which side it is_ feels like the modifier.

Reply:
addressed in [2e541ad](https://github.com/ithaka/pharos/pull/608/commits/2e541ad604a4cce7e0990e07409136c19bbaf632)